dev/tests/integration/testsuite/Magento/Catalog/Model/Indexer/Category/ProductTest.php

```diff
@@ -184,7 +184,36 @@ public function testCategoryDelete()
         }
     }
 
-    public function testCategoryCreate()
+
+    /**
+     * Verify that indexer still valid after deleting inactive category
+     *
+     * @magentoAppArea adminhtml
+     * @magentoDataFixture Magento/Catalog/_files/categories_disabled.php
+     *
+     * @return void
+     */
+    public function testDeleteInactiveCategory(): void
+    {
+        $this->indexer->reindexAll();
+        $indexerShouldBeValid = $this->indexer->isInvalid();
```

Comment:
Hello @engcom-Charlie thanks for your contribution!

The name of `$indexerShouldBeValid` variable wasn't very intuitive to me, it didn't give me the idea that we were getting the state of the indexer.

What about renaming with something like `$isInvalidIndexer`?

This way the assertion `$this->assertFalse($isInvalidIndexer)` becomes more readable too.

What do you think?